### PR TITLE
Resolve Technical Debt and update acceptance tests follow the new changes in the FIWARE Catalogue

### PR DESCRIPTION
**Pull request by flopezag**
### REVIEWERS

@chemaper @jframos 
### DESCRIPTION
- Resolution of some Technical Debt.
- Update Acceptance test associated to getnid due to an update in the FIWARE Catalogue
### TESTING

Acceptance test of the getnid.

### CODE REVIEW

**Comment by jframos** [comment]
This branch has conflicts with _develop_.

**Comment by jframos** [comment]
I would prefer not to convert `tests/acceptance/` in a python module, addiding a new __init__.py.
The same with the `tests/` folder.

File: scripts/support/create_ge_images/publish_image.py [hunk 1]
```diff
@@ -28,11 +28,13 @@
...
-        '<image_uuid> [oldname]'
-    print >> sys.stderr, '  otherwise          : ', sys.argv[0], \
-        '<image_uuid> <nid> <type>'
+    sys.stderr.write('Please, use:')
+
+    msg = '  If the image exists: {} <image_uuid> [oldname]'.format(sys.argv[0])
+    sys.stderr.write(msg)
+
+    msg = '  otherwise          : {} <image_uuid> <nid> <type>'.format(sys.argv[0])
+    sys.stderr.write(msg)
```
Comment (chemaper) [inline scripts/support/create_ge_images/publish_image.py:38]: Every write must have + '\n' to be  equivalent to print

File: scripts/support/create_ge_images/publish_image.py [hunk 2]
```diff
@@ -40,8 +42,8 @@
 glance = osclients.get_glanceclient()
 image = glance.images.get(sys.argv[1])
 if image.name[-3:] != '_rc':
-    msg = 'According the name, the image to publish is not a _rc:'
-    print >>sys.stderr, msg, image.name
+    msg = 'According the name, the image to publish is not a _rc: {}'.format(image.name)
+    sys.stderr.write(msg)
     sys.exit(-1)
```
Comment (chemaper) [inline scripts/support/create_ge_images/publish_image.py:47]: Every write must have + '\n' to be  equivalent to print

File: scripts/support/create_ge_images/publish_image.py [hunk 3]
```diff
@@ -59,21 +61,20 @@
...
+            sys.stderr.write(m)
+            sys.stderr.write(i.name, i.id, i.owner)
             continue
         if i.owner != owner:
             m = 'Warning: image with the same name found, with another owner'
-            print >>sys.stderr, m
-            print >>sys.stderr, i.name, i.id, i.owner
+            sys.stderr.write(m)
+            sys.stderr.write(i.name, i.id, i.owner)
             continue
```
Comment (chemaper) [inline scripts/support/create_ge_images/publish_image.py:74]: Every write must have + '\n' to be  equivalent to print

**Comment by chemaper** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**